src/mme-app/mmeStates/defaultMmeState.cpp

```diff
@@ -106,11 +106,6 @@ void DefaultMmeState::initialize()
                 actionTable.addAction(&ActionHandlers::default_create_bearer_req_handler);
                 eventToActionsMap.insert(pair<uint16_t, ActionTable>(CREATE_BEARER_REQ_FROM_GW, actionTable));
         }
-        {
```

Comment:
Why no default state for DDN fail event?

Reply:
For UE Init Service Request, we are sending SERVICE REJECT and not cleaning up the UE Context. 
Followed the same for DDN/Paging Init Service Request as well.

Please clarify if we need to change this.